src/Catty/Helpers/Util/Util.swift

```diff
@@ -50,6 +50,16 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         return appBuildVersion
     }
 
+    class func alert(withText text: String?) {
+        alert(withTitle: kLocalizedPocketCode, andText: text)
+    }
+
+    class func alert(withTitle title: String?, andText text: String?) {
```

Comment:
Please make the parameters non-optional. Also, I would suggest to change the Swift method signature to `alert(title: String, text: String)` (ObjC should remain `alertwithTitle: andText:`).

Reply:
See comment above.

Reply:
Yes, please change it.

Reply:
Attributes have been added, works fine 👌🏼